packages/grafana-ui/src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,82 @@
+import { css } from '@emotion/css';
+import { FocusScope } from '@react-aria/focus';
+import React, { useState } from 'react';
+import { usePopperTooltip } from 'react-popper-tooltip';
+import { CSSTransition } from 'react-transition-group';
+
+import { ReactUtils } from '../../utils';
+import { Portal } from '../Portal/Portal';
+import { TooltipPlacement } from '../Tooltip/types';
+
+export interface Props {
+  overlay: React.ReactElement | (() => React.ReactElement);
+  placement?: TooltipPlacement;
+  children: React.ReactElement;
+  /** Defaults to click */
+  trigger?: Array<'click' | 'hover'>;
```

Comment:
**Nit:** maybe we could define our own type for this trigger and use it in `Tooltip` as well

E.g.:
```javascript
// grafana-ui/src/components/Tooltip/types.ts
export type TooltipTrigger = 'click' | 'hover' ;

// ...

// grafana-ui/src/components/Dropdown/Dropdown.tsx
trigger?: TooltipTrigger | TooltipTrigger[]
```

Reply:
We don't have a trigger propery for Tooltip component. But could add a type for it for this Dropdown as a DropdownTrigger ?

Reply:
@leventebalogh removed the trigger property as supporting hover and focus at the same time was a bit tricky and we have no current pressing use cases for hover right now.

Reply:
Sorry for the late answer @torkelo. Sure, that sounds like a good point 👍